packages/shared/providers.yaml

```diff
@@ -2721,7 +2721,7 @@ freshteam:
             description: The API key to your Freshteam account
             doc_section: '#step-2-finding-your-api-key'
             example: 'XtoF94LMNyKPbd5AvR1QJh'
-            pattern: '^[a-zA-Z0-9]+$'
+            pattern: '^[a-zA-Z0-9_!@#$%^&*()\\-+=]+$'
```

Comment:
what kind of api key is this? 🤔

Reply:
Not sure, just the client reported that our regex was too restrictive for their API key. The provider docs don't have any info on the structure unfortunately.

Reply:
would have been nice to confirm with the customer because a too broad validation is not very useful